introduction.mdx

```diff
@@ -14,36 +14,29 @@ description: 'Welcome to Aryn!'
+* **Powerful and accurate document segmentation for complex documents with tables, images, text, infographics, and more.**  DocParse's vision-based segmentation models provide 11+ class labels, and are up to 6x better in mean average precision (mAP) and 4x better in mean average recall (mAR) than other parsing solutions.
 
-* **Plug-and-Play LLMs.** Use different LLMs for entity extraction, vector embedding, and post-processing steps.
+* **AI-powered table extraction, OCR, image processing, and chunking.** Use DocParse's best-in-class table extraction and optical character recgonition (OCR) features to unlock data in your documents. Take the structured output of DocParse and leverage DocPrep's chunking strategies to experiment with the best sizing of chunks for your use case.
 
+* **Scalable and reliable loading of vector DBs and search indexes.** DocPrep's ETL pipelines can generate vector embeddings using your choice of embedding model and load a vareity of target databases. These include leading engines like Elasticsearch, OpenSearch, Weaviate, Pinecone, DuckDB, Qdrant, and more. DocPrep's ETL pipelines can scale from one to thousands of documents.
 
-## Aryn DocParse
-
-You can use Aryn DocParse to segment documents into labeled bounding boxes identifying titles, tables, table rows and columns, images, and regular text. The service can segment PDFs, Microsoft Word (.docx and .doc) documents, and Microsoft PowerPoint (.pptx and .ppt) file formats. It is often used to [extract structured table data](docparse/tables). Bounding boxes are returned as JSON with their associated text for easy use. It leverages [Aryn's purpose-built AI model for document segmentation and labelling](https://huggingface.co/Aryn/deformable-detr) that was trained using DocLayNet - an open source, human-annotated document layout segmentation dataset containing tens of thousands of pages from a broad variety of document sources.
-
-
-If you'd like to experiment with the service, you can use the UI in the [Aryn Playground](https://play.aryn.cloud/partitioning) to visualize how your documents will be partitioned. To get started with Aryn DocParse, please refer to the [Quickstart](quickstart) guide.
+* **Fully customizable document ETL in Python.** DocPrep generates document ETL pipeliens in Python using the open source [Sycamore document ETL library](https://github.com/aryn-ai/sycamore). Customize your pipeline with additional data transforms, LLM-baed entity extraction, data enrichment, and data cleaning.
 
 
 ## Setting up
 
-[Sign-up here for free](https://www.aryn.ai/get-started) to get started with an API Key. Note, this is the documentation for Aryn DocParse. For Sycamore, please visit the [Sycamore documentation](https://sycamore.readthedocs.io/en/stable/).
```

Comment:
Are we sure we don't want to link to Sycamore docs at all? It will only be visible in the left sidebar then.

Reply:
I put a link in the full text. We don't need more links for now, and we can add once we get a full view.